### PR TITLE
Adds a bridge betwen Cripts and HRW operators

### PR DESCRIPTION
**Pull request by zwoop**
This also adds some new mechanisms to some Cripts core, and updates documentation as well. 

Note: This does not handle all HRW conditions yet, rather, I think we can add them as people have need for them.

### CODE REVIEW

File: src/cripts/Bundles/Headers.cc [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+namespace Bundle
+{
+const Cript::string Headers::_name = "Bundle::Headers";
+
+bool
+Headers::validate(std::vector<Cript::Bundle::Error> &errors) const
+{
+  bool failed = false;
+  // errors.emplace_back("dscp must be between 0 and 63", name(), "dscp");
+  // failed = true;
```
Comment (cmcfarlen) [inline src/cripts/Bundles/Headers.cc:60]: This function seems unfinished.

**Review by cmcfarlen — APPROVED** [review]

File: src/cripts/Bundles/Headers.cc (hunk 1 above)
Comment (zwoop) [reply src/cripts/Bundles/Headers.cc]: fixed / removed, it was a remnant.

**Comment by zwoop** [comment]
[approve ci autest]

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**